server/src/main/java/org/schoellerfamily/pathfinderhelper/datamodel/Character.java

```diff
@@ -0,0 +1,337 @@
+package org.schoellerfamily.pathfinderhelper.datamodel;
+
+import java.util.ArrayList;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@Entity
+public class Character {
+
+    @Id
+    @GeneratedValue
+    private Long id;
+    private String name;
+    private Long playerid;
+    private Integer strength;
+    private Integer dexterity;
+    private Integer constitution;
+    private Integer intelligence;
+    private Integer wisdom;
+    private Integer charisma;
```

Comment:
With so many different attributes, this might best be supported with a builder pattern. There are 2 variations of this:
1. The class supports its own builder behaviors.
2. You create a separate builder class.